.github/workflows/release-artifacts.yml

```diff
@@ -200,10 +200,10 @@ jobs:
             nix run github:ComposableFi/composable/v${{ env.COMPOSABLE_VERSION }}#composable-node-release
 
             # Spin up a local devnet (Dali + Rococo)
-            nix run github:ComposableFi/composable/v${{ env.COMPOSABLE_VERSION }}#devnet-up
+            nix run github:ComposableFi/composable/v${{ env.COMPOSABLE_VERSION }}#devnet
 
             # Spin up a local XCVM devnet (Dali + Rococo + Juno)
-            nix run github:ComposableFi/composable/v${{ env.COMPOSABLE_VERSION }}#devnet-xcvm-up
+            nix run github:ComposableFi/composable/v${{ env.COMPOSABLE_VERSION }}#devnet-xcvm
 
             # Show all possible apps, shells and packages
             nix flake show github:ComposableFi/composable/v${{ env.COMPOSABLE_VERSION }} --allow-import-from-derivation
```

Comment:
We would also like the release artifacts for wasm to include the runtime spec_version, according to the pattern

vA.B.C -> picasso-ABC 
Eg:
v2.4.2 -> picasso-242

Reply:
We will make sure that this is the actual spec_version by hand for now

Reply:
But we can automate later